packages/braid-design-system/src/lib/components/private/Popover/Popover.test.tsx

```diff
@@ -0,0 +1,240 @@
+import '@testing-library/jest-dom';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { useRef, useState } from 'react';
+
+import { BraidTestProvider } from '../../BraidTestProvider/BraidTestProvider';
+import { Button } from '../../Button/Button';
+import { Placeholder } from '../Placeholder/Placeholder';
+
+import { Popover } from './Popover';
+
+function renderPopover({
+  open = false,
+  useManagedState = false,
+  useEnterFocusRef = false,
+}: {
+  open?: boolean;
+  useManagedState?: boolean;
+  useEnterFocusRef?: boolean;
```

Comment:
I know its internal, but suggest renaming to avoid confusing with the Hook naming convention